packages/create-react-app/createReactApp.js

```diff
@@ -275,6 +286,10 @@ function install(root, useYarn, dependencies, verbose, isOnline) {
       args.push('--verbose');
     }
 
+    if (binLinks === false) {
```

Comment:
I don't understand this conditon. Should we not be checking that the binLinks option was passed before adding the argument?

Reply:
Hey @bondz !

Great question!  I was trying to follow the same pattern that you use for --use-npm and --verbose, so I was passing the option down the same way that they were.  The reason for checking for it to be false is because of the way commander checks args; if any arg is passed with the --no flag, it will set that process to false by default.

So the way it was requested, it looks like the request was to be able to pass an option to create-react-app for --no-bin-links.  So if that option is passed, then it will set process.binLinks to false (which is why I specifically checked for it to be false).

Reply:
Oh, okay. I understand now. Thanks for clarifying.

Reply:
No problem!  Just let me know if I can help or explain anything else!